superdesk/io/feeding_services/rss.py

```diff
@@ -38,13 +38,46 @@ class RSSFeedingService(FeedingService):
     """
 
     NAME = 'rss'
+
     ERRORS = [IngestApiError.apiAuthError().get_error_description(),
               IngestApiError.apiNotFoundError().get_error_description(),
               IngestApiError.apiGeneralError().get_error_description(),
               ParserError.parseMessageError().get_error_description()]
 
     label = 'RSS'
 
+    fields = [
+        {
+            'id': 'url', 'type': 'text', 'label': 'Host',
+            'placeholder': 'RSS Feed URL', 'required': True,
+            'errors': {4001: 'Connection timed out.', 4006: 'URL not found.',
+                       4009: 'Can\'t connect to host.', 1001: 'Can\'t parse the RSS.'}
+        },
+        {
+            'id': 'auth_required', 'type': 'boolean', 'label': 'Requires Authentication',
+            'placeholder': 'Requires Authentication', 'required': False
+        },
+        {
+            'id': 'username', 'type': 'text', 'label': 'Username',
+            'placeholder': 'Username', 'required_expression': '{auth_required}',
+            'show_expression': '{auth_required}'
+        },
+        {
+            'id': 'password', 'type': 'password', 'label': 'Password',
+            'placeholder': 'Password', 'required_expression': '{auth_required}',
+            'show_expression': '{auth_required}'
+        },
+        {
+            'id': 'field_aliases', 'type': 'tuple-multivalue', 'label': 'Content Field Aliases',
```

Comment:
what about something like `type='mapping'` and instead of `first_field_label` having "recursive" fields:
```
{
    'label': 'Content Field Aliases',
    'type': 'mapping',
    'fields': [
        {'type': 'select', 'label': 'RSS field name', 'options': [...]},
        {'type': 'text', 'label': 'Superdesk field'},
    ],
}
```
I think this would make the api a bit cleaner and wouldn't require ui changes when we need more columns